wordpress-mobile/WordPress-iOS
# Adds tracking when expanding items on the Stats: Post Stats view

Project: https://github.com/wordpress-mobile/WordPress-iOS/issues/17503
Depends on WPiOS Shared PR: https://github.com/wordpress-mobile/WordPress-iOS-Shared/pull/303

### To test:

1. Launch the app
2. Tap on the My Site tab
3. Tap on the 'Stats' item
4. Tap on an item under 'Posts and Pages' to open the 'Post Stats' view
5. Under Months and Years tap the down arrow to expand a section
6. Verify you see: `🔵 Tracked: stats_posts_and_pages_months_years_item_tapped <blog_id: BLOG_ID, site_type: SITE_TYPE>`
5. Under Recent Weeks tap the down arrow to expand a section
6. Verify you see: `🔵 Tracked: stats_posts_and_pages_recent_weeks_item_tapped <blog_id: BLOG_ID, site_type: SITE_TYPE>`

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
None

3. What automated tests I added (or what prevented me from doing so)
None

PR submission checklist:
